Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -188,7 +188,14 @@ extension BrowserViewController: WKNavigationDelegate {
             decisionHandler(.cancel)
             return
         }
+        
         let isPrivateBrowsing = PrivateBrowsingManager.shared.isPrivateBrowsing
+        
+        if url.baseDomain == "youtube.com" {
```

Comment:
the WKNavDelegate getting bigger and bigger :(

Nothing to do now, but if we add another site specific hack for adblocking we should abstract it somewhere, maybe in Tab or in userScriptManager itself